x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecks.java

```diff
@@ -79,23 +79,6 @@ private static void fieldLevelMappingIssue(IndexMetaData indexMetaData, BiConsum
         return issues;
     }
 
-    static DeprecationIssue coercionCheck(IndexMetaData indexMetaData) {
```

Comment:
++ I take it this is being removed because it is a user of `INFO` and the other stale checks will be removed in follow-ups?

Reply:
Yes, this is the only current user of `INFO`. The other stale checks should also be removed, but I figure that's another PR.